File: Sound.pas
Program Sound;
Const 
   Freq = 250;
   SIM = 60;
Var 
   t, a, r : Real;
Begin
   a := Freq / SIM;
   ReadLn(t);
   r:= a * t;
   WriteLn(r);
End.

{ Code by GPU Officer Yahwe, Toverisch Trunaev, Russia } 

File: Sound.pl
#!/usr/bin/perl

use 5.022;

my $freq = 250.0;
my $SIM = 60.0;
my ($t, $a, $r);

$a = $freq / $SIM;
$t = <>;
$r = $a * $t;
 
say $r;

# Code by GPU Officer Yahwe
